PSReadLine/Cmdlets.cs

```diff
@@ -825,6 +825,10 @@ public SwitchParameter Unbound
         }
         private SwitchParameter? _unbound;
 
+        [Parameter]
+        [Alias("Key")]
+        public string[] Chord { get; set; }
```

Comment:
This should be put in a separate parameter set. When specifying `-Chord`, it implies searching from the bound bindings.
After adding this parameter, the syntax should look like below, and the `[-Bound] [-Unbound]` parameter set should be made the default.
```
PS:82> gcm Get-PSReadLineKeyHandler -Syntax

Get-PSReadLineKeyHandler [-Bound] [-Unbound] [<CommonParameters>]

Get-PSReadLineKeyHandler -Chord [<CommonParameters>]
```

Reply:
I've added it to a separate ParameterSet, made it mandatory for that ParameterSet, and allowed it to be positional.